label_studio/ml/models.py

```diff
@@ -30,6 +30,11 @@ class MLBackendState(models.TextChoices):
     PREDICTING = 'PR', _('Predicting')
 
 
+class MLBackendAuth(models.TextChoices):
+    NONE = 'NA', _('None')
+    BASIC_AUTH = 'BA', _('Basic Auth')
```

Comment:
We save almost nothing by using two-letter codes in this charfield, why not make the CharField len 255 and make the enum values the same as the enum name? This means nobody ever has to figure out that "NA" is the same as NONE by reading backend code, for example.